resume.html

```diff
@@ -25,10 +25,10 @@ <h1 class="line">Resume</h1>
             <div class="wrapper">
                 <div class="resume-box">
                     <h2>Senior Principal UX Designer</h2>
-                    <div class="jobs-info2">I am a multi-platform experience designer with over 12 years of experience. My background includes a degree in business marketing, art direction in direct advertising and an MFA in Media Design prior to my focus on interactive experiences. I am a system thinker and get most excited about brainstroming and creating strategic solutions with designers, product and engineer partners. Designing and developing an experience that is logical, efficient and perhaps a bit serenditpitous is exciting and rewarding. I design all phases of a project including end-to-end product design.</div>
+                    <div class="jobs-info2">I am a multi-platform, end-to-end experience designer with over twelve years of experience. My background includes a business degree in marketing, art direction in direct advertising and an MFA in Media Design prior to my focus on interactive experiences.</div>
                     <br/>
 
-                    <button><a href="#">Download PDF</a></button>
+                    <button><a href="images/Jen-Byyny-resume.pdf" target='blank'>Download PDF</a></button>
```

Comment:
sorry, i made a mistake here before. it should be `<a href="images/Jen-Byyny-resume.pdf" target='blank'><button>Download PDF</button></a>` so that the link wraps the entire button.